WordPress/Classes/Utility/CoreData/ContextManager.swift

```diff
@@ -234,13 +235,14 @@ private extension ContextManager {
             fatalError("Can't create object model named \(modelName) at \(modelFileURL)")
         }
 
-        let startupEvent = SentryStartupEvent()
+        // FIXME: Import the Sentry stuff, too — But it accesses the app delegate!
+        // let startupEvent = SentryStartupEvent()
```

Comment:
@crazytonyli is this `ContextManager` Sentry logging something that we need, or is it by any chance something that we added at the time of adoption just to monitor 🤞 ?

Being able to just chop it off would simplify things.

Reply:
See also p1743115991824589/1743090023.463689-slack-C08HQR4C2TS

Reply:
I'm sorry I missed this message. I don't think I added the events. It's very likely that they were there before my time. I doubt these events ever occur on production. I did a quick search on Sentry and don't see any related events. However, I think it's best to keep them in case we introduce some issues to the Core Data layer. At the bare minimum, we can send the events to the app via a notification.